server/src/metadata.rs

```diff
@@ -48,6 +48,26 @@ pub mod entities_metadata_query {
         entities: Vec<EntityMetadata>,
     }
 
+    #[derive(Serialize)]
+    enum CreatorType {
+        Person,
+        // Should we support this type too?
+        // Organization,
+    }
+
+    #[derive(Serialize)]
+    #[serde(rename_all = "camelCase")]
+    struct Creator {
+        #[serde(rename = "type")]
+        creator_type: CreatorType,
+        id: Option<String>,
+        name: String,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        affiliation: Option<serde_json::Value>,
```

Comment:
```suggestion
        affiliation: Option<serde_json::Value>,
```

`affiliation` can be Serlo:

```json
{
  "affiliation": {
    "id": "https://serlo.org/",
    "type": "Organization",
    "name": "Serlo Education e.V."
  }
}
```

(JSON should be the same as in `provider`)